### ISSUE TITLE
Update Installation for IPython 3.0

### ISSUE THREAD

**Issue by ssanderson** [quantopian/qgrid#17]
Will fix issues like https://github.com/quantopian/qgrid/issues/16.

**Comment by signalseeker** [comment]
Any idea how to install this for ipython 3.0?

I tried pip install git+https://github.com/quantopian/qgrid but I get the following error -

```python UnknownExtra: ipython 3.0.0 has no such extra feature 'notebook' ```

**Comment by chausler** [comment]
I'm also seeing this issue. Any work arounds?

**Comment by TimShawver** [comment]
@chausler @signalseeker I spent some time looking into this and haven't been able to reproduce the issue.  How did you install IPython 3.0? I attempted to reproduce it by installing IPython 3.0 like this:
```
pip install "ipython[all]"==3.0.0
```
Running `pip install git+https://github.com/quantopian/qgrid` completes successfully for me after that.  From the error message it sounds like the installation is getting tripped up by the `ipython[notebook]` requirement in our setup.py file, which seems strange to me because IPython 3.0.0 definitely has the notebook "extra".
